### DISCUSSION TITLE
Show logs from daemonized processes?

### DISCUSSION THREAD

**Discussion by neezer** [F1bonacc1/process-compose#281] [Q&A] [answered]
Is it possible to show logs in `process-compose` for background (`is_daemon: true`) processes? I'm specifically wondering if I can [A] pipe the output of `journalctl` for a systemd process or [B] tail a log file into/in `process-compose`... I didn't see anything in the docs that offered this functionality.

**Comment by F1bonacc1** [comment, marked as answer]
Hey @neezer,
I would like to understand your use case a bit better.
Is it possible to start the process in a non-daemon mode?
Will creating an additional process that `tail -f` the log or journalctl work in your scenario?

**Reply by neezer** [reply]
> Is it possible to start the process in a non-daemon mode?

Yes, and this is probably the best thing to do. I feel like an idiot for not just copying the `ExecStart` commands out of the systemd file and using those directly in `process-compose`. That works great, so I'm just going to roll with that now. Thanks for the help!
